ChangeLog

```diff
@@ -4,6 +4,106 @@ brevity.  Much more detail can be found in the git revision history:
 
     https://github.com/jemalloc/jemalloc
 
+* 5.3.0 (May TBD, 2022) ###TODO update date
```

Comment:
Just in case - don't forget to change it.

Reply:
I'll make sure to update before merging. Still waiting for Travis CI to get back, which may take a day or two.